book/intro.md

```diff
@@ -1,11 +1,207 @@
-# Welcome to your Jupyter Book
+# Mastering Geospatial Visualizations with GMT/PyGMT
 
-This is a small sample book to give you a feel for how book content is
-structured.
-It shows off a few of the major file types, as well as some sample content.
-It does not go in-depth into any particular topic - check out [the Jupyter Book documentation](https://jupyterbook.org) for more information.
+Welcome to the AGU24 [GMT](https://docs.generic-mapping-tools.org/6.5)/
+[PyGMT](https://www.pygmt.org/v0.13.0) workshop 🥳! This Jupyter book 📖 contains
+tutorials for making maps 🗺️ and animations 🎦
 
-Check out the content pages bundled with this sample book to see more.
+::::::{grid} 1 1 3 3
+:gutter: 1
+:padding: 1
 
-```{tableofcontents}
+
+:::::{grid-item}
+::::{grid} 1 1 1 1
+:gutter: 1
+
+:::{grid-item-card} Tutorial 1 - First figure + Subplots / layout
+:img-top: _images/fd70248f75b8b37ee54b3135f77705f98c6d8489eb18b6910184b447d0f0638d.png
+:link: ./tut01_firstfigure.html
+by [Jing-Hui Tong](https://orcid.org/0009-0002-7195-3071)
++++
+{bdg-primary}`pygmt`
+{bdg-primary-line}`coast`
+{bdg-primary-line}`colorbar`
+{bdg-primary-line}`grdimage`
+{bdg-primary-line}`makecpt`
+{bdg-primary-line}`subplot`
+{bdg-secondary-line}`earth_relief`
+{bdg-secondary-line}`earth_age`
+:::
+
+:::{grid-item-card} Tutorial 2 - Integration with the Scientific Python Ecosystem: Pandas / GeoPandas
```

Comment:
```suggestion
:::{grid-item-card} Tutorial 2 - Integration with the scientific Python ecosystem: pandas / GeoPandas
```

Reply:
I guess @weiji14 try to maintain consistency across all titles by capitalizing the first letter of each word throughout the page.

Reply:
I'll take @yvonnefroehlich's suggestions on capitalization, we should try to match with the titles on the left side bar too.